scripts/locales/en.lua

```diff
@@ -12,6 +12,16 @@ local  en = {
    difference = "Difference",
    total = "Total",
    today = "Today",
+   NOTES = "NOTES",
```

Comment:
maybe we will add a "Notes" string in the future (only first letter capitalized), so I want to distinguish this here. As a workaround, I can place a notes -> "NOTES" entry under "shaping"